django_mongodb/client.py

```diff
@@ -1,14 +1,51 @@
 import signal
+import urllib
 
 from django.db.backends.base.client import BaseDatabaseClient
 
 
 class DatabaseClient(BaseDatabaseClient):
-    executable_name = "mongo"
+    executable_name = "mongosh"
+    help_options = ("--help", "-h")
 
     @classmethod
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
-        raise NotImplementedError
+        options = settings_dict.get("OPTIONS", {})
+        args = [cls.executable_name]
+
+        host = settings_dict["HOST"]
+        port = settings_dict["PORT"]
+        dbname = settings_dict["NAME"] or "test"
```

Comment:
Is "test" needed? When I execute `monogsh` without parameters, it seems to connect to `test` automatically. If it's needed, then it seems `if dbname` later on will always be true.

Reply:
Yes, It was on purpose.
if I don't define test when I call `python3 manage.py dbshell filename`  instead of use filename as a script it will connect to the database filename. because some commands like this:
`mongosh filename` is made. Then mongosh thinks that filename is the database.
And the second thing yes, the if that is below has no effect now. Will remove